CAFAna/Prediction/IPrediction.cxx

```diff
@@ -28,18 +29,24 @@ namespace ana
     const TString tag = ptag->GetString();
     delete ptag;
 
+    if(tag == "PredictionNonSwapNoExtrap") return PredictionNonSwapNoExtrap::LoadFrom(dir, name);
+    if(tag == "DataPredictionNoExtrap") return DataPredictionNoExtrap::LoadFrom(dir, name);
```

Comment:
This is one point against moving the PRISM-specific predictions out of Core/. NOvA has a much better mechanism for registering LoadFroms this that would allow that move.